web/ui/card-link.svelte

```diff
@@ -17,14 +17,14 @@
   }
 </script>
 
-<li role="presentation">
+<li role="none">
```

Comment:
Why do we have this change?

Reply:
- "presentation" and "none" are synonyms
- "none" is shorter

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/none_role

To be more representative - I think role="none" is more useful since from the naming you didn't expect it will be somehow iterate with UI/UX, while naming presentation - make give you false expectations

Reply:
Good idea